index.test.js

```diff
@@ -58,3 +58,51 @@ test('create pr', async t => {
 
   t.same(response.statusCode, 200)
 })
+
+test('create release', async t => {
+  const mockedRepo = { repo: 'smn-repo', owner: 'salmanm' }
```

Comment:
is there an existing test which checks that this works if a target is NOT provided? I assume so. If not, make sure to create one. If it does, let's give this test a more meaningful name and maybe group together those two tests so we're colocating the two cases: one with and one without the target

Reply:
Yep, this test was missing, just added it.